src/python/components/AppPicker.py

```diff
@@ -1,8 +1,9 @@
 import importlib
-import os
-
-from util import getChild
+import logging, os
 import streamlit as st
+from util import getChild
+
+logging.basicConfig(format="%(levelname)s %(asctime)s %(name)s:%(message)s\n")
```

Comment:
regression? `basicConfig()`  should be in util

Reply:
Oops, I moved that there during my testing and forgot to move it back. Good catch, thanks!